.gitmodules

```diff
@@ -6,7 +6,7 @@
 	url = https://github.com/daveshah1/up5k-demos.git
 [submodule "third_party/vexriscv-verilog"]
 	path = third_party/vexriscv-verilog
-	url = https://github.com/m-labs/VexRiscv-verilog
+	url = https://github.com/antmicro/VexRiscv-verilog
```

Comment:
I saw from the previous CI that the only failing test was the baselitex nextpnr-xilinx one. By looking at the error I noticed that the actual failure is related to DSPs which are not fully supported in symbiflow at the moment. Probably it would be better to ass a synthesis flag when calling yosys for nextpnr-xilinx to disable dsp inference (`-nodsp`): https://github.com/SymbiFlow/fpga-tool-perf/blob/0b0d22a5f79f11c9e84325becef51944e98ba1f3/symbiflow.py#L661-L663